bigquery/docs/snippets.py

```diff
@@ -13,11 +13,9 @@
 # limitations under the License.
 
 """Testable usage examples for Google BigQuery API wrapper
-
```

Comment:
This and a few other similar blank lines should probably not be deleted?

By convention the docstring title line should be separated from the body with a blank line. Or was there some kind of a linter check that complained about it?

Reply:
`snippets.py` file is gonna be deprecated. Maybe I deleted this string accidentally. And didn't notice it. If Tim will find it critical, I put it back.

Reply:
I see. If the file is going to be deprecated, then this is probably indeed a lesser concern